modules/apps/object/object-test/src/testFunctional/tests/ObjectFields.testcase

```diff
@@ -421,12 +421,34 @@ definition {
 	}
 
 	@description = "LPS-148112 - Verify if when selecting the option on Request Files: Directly from User's Computer there is a Toggle"
-	@ignore = "Test Stub"
 	@priority = "4"
 	test CanViewOptionsWhenSelectDirectlyFromUsersComputerOption {
+		ObjectAdmin.addObjectViaAPI(
+			labelName = "Custom Object",
+			objectName = "CustomObject",
+			pluralLabelName = "Custom Objects");
 
-		// TODO LPS-153411 CanViewOptionsWhenSelectDirectlyFromUsersComputerOption pending implementation
+		Navigator.openURL();
+
```

Comment:
See https://github.com/timpak/liferay-portal/pull/252/commits/e8bf3ac3a6c2da3bc94d23b3a650b11449666610

Since the next macro is opening a specific page, we don't need to navigate back to the home page here.